app/src/main/java/ru/prometeydev/movie/ui/movieslist/MoviesListNavigable.kt

```diff
@@ -0,0 +1,18 @@
+package ru.prometeydev.movie.ui.movieslist
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import ru.prometeydev.movie.ViewModelProviderFactory
+import ru.prometeydev.movie.common.show
+import ru.prometeydev.movie.data.Movie
+import ru.prometeydev.movie.ui.moviesdetails.MoviesDetailsFragment
+
+open class MoviesListNavigable : Fragment() {
```

Comment:
Same, I don't understand why do you need this base class

Reply:
I wanted to do some kind of abstraction, but it turned out not beautiful, probably I will remove it, I will correct